endaq/device/schemata/command-response.xml

```diff
@@ -143,21 +158,28 @@
         </UIntegerElement>
 		<BinaryElement name="PingReply" id="0x5701" multiple="0" mandatory="0">Serial only. Contains the echo to a Ping command.</BinaryElement>
         <IntegerElement name="DeviceStatusCode" id="0x5801" multiple="0" mandatory="0">Serial only, mandatory for serial. int16. Negative values are errors.
+            40: Device Triggering (i.e. waiting for a trigger)
             30: Recording start pending (device will start recording soon after this message is sent)
             20: Reset pending (device will reset soon after this message is sent)
             10: Device recording
             0: Device idle, message successful.
             -10: Communication busy
             -20: Command not valid
             -21: Command Lock ID invalid
+            -22: Command Info Index Unknown / Invalid
             -30: Unknown command
             -40: Bad payload
             -50: EBML invalid
+            -51: Response too large, internal device error
             -60: Bad checksum
             -70: Preamble (CorBus packet header) invalid
         </IntegerElement>
         <UnicodeElement name="DeviceStatusMessage" id="0x5901" multiple="0" mandatory="0">Serial only. Optional code for device status, including error message</UnicodeElement>
         <BinaryElement name="LockID" id="0x5A00" mandatory="0" multiple="0"> Serial Only, optional current LockID of device </BinaryElement>
+        <MasterElement name="GetInfoResp" id="0x5B01" mandatory="0" multiple="0" >
```

Comment:
`<GetInfoResp>` should be spelled out in full (`GetInfoResponse`). No other response abbreviates it.